From cb9759dc5daf1ad92d922be205d02cc10ed34c93 Mon Sep 17 00:00:00 2001
From: Radu Pirea <radu-nicolae.pirea@nxp.com>
Date: Mon, 1 Mar 2021 12:00:48 +0200
Subject: [PATCH 03/42] configs: bluebox3: disable dram inline ecc

Dram inline ECC initialization takes too long and there is no enough
time to disable the PMIC watchdog.

Upstream-Status: Pending

Issue: ALB-6512

Signed-off-by: Radu Pirea <radu-nicolae.pirea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 configs/s32g274abluebox3_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/s32g274abluebox3_defconfig b/configs/s32g274abluebox3_defconfig
index 674c01bd9a..b9d2ad4d80 100644
--- a/configs/s32g274abluebox3_defconfig
+++ b/configs/s32g274abluebox3_defconfig
@@ -5,6 +5,7 @@ CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G274ABLUEBOX3=y
 CONFIG_S32_LPDDR4=y
+# CONFIG_S32GEN1_DRAM_INLINE_ECC is not set
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_SYS_MALLOC_F is not set
 CONFIG_TOOLS_DEBUG=y
-- 
2.25.1

